src/options.rs

```diff
@@ -57,18 +57,26 @@ impl FromStr for Sanitizer {
 
 #[derive(Clone, Debug, StructOpt)]
 pub struct BuildOptions {
-    #[structopt(short = "O", long = "release")]
+    #[structopt(short = "D", long = "debug", conflicts_with = "release")]
+    /// Build artifacts in debug mode, without optimizations
+    pub debug: bool,
+
+    #[structopt(short = "O", long = "release", conflicts_with = "debug")]
     /// Build artifacts in release mode, with optimizations
     pub release: bool,
 
-    /// Build target with verbose output from `cargo build`
-    #[structopt(short = "v", long = "verbose")]
-    pub verbose: bool,
-
     #[structopt(short = "a", long = "debug-assertions")]
     /// Build artifacts with debug assertions enabled (default if not -O)
     pub debug_assertions: bool,
 
+    #[structopt(short = "o", long = "overflow-checks")]
+    /// Build artifacts with overflow checks enabled (default if not -O)
```

Comment:
Regarding "default if not -O": right now, passing `-Oa` will enable overflow checks, is that not the case anymore after this PR? Many projects use `-Oa` and I'd be wary of making it lose the overflow checks. Do `debug-assertions` imply `overflow-checks` or are they disconnected now?

Reply:
> right now, passing -Oa will enable overflow checks, is that not the case anymore after this PR?

That is still the case. The way these flags work is unchanged. Only the default state is altered, which is now equivalent to `-Ooa`

> Do debug-assertions imply overflow-checks or are they disconnected now?

I believe debug-assertions never implied overflow-checks, and enabling overflow-checks in release mode was not possible prior to this PR.

Reply:
Oh wait, nevermind, I was confused. I'll have to test whether `-a` actually enables overflow checks or not.

Reply:
Okay, I've tested it and turns out `--debug-assertions` already enables overflow checks, so there's no real need to have a separate flag for it. I'll drop the extra flag.